astropy/io/fits/header.py

```diff
@@ -687,7 +693,8 @@ def tofile(self, fileobj, sep='', endcard=True, padding=True,
 
     @classmethod
     def fromtextfile(cls, fileobj, endcard=False):
-        """
+        """Read a header from a simple text file or file-like object
```

Comment:
Missing a `.` at the end of the sentence. I would also include a link to `Header.fromfile` here.